powershell-adapter/Tests/PSTestModule/1.0.0/DscResources/TestPSRepository/TestPSRepository.psm1

```diff
@@ -14,7 +14,7 @@ function Get-TargetResource {
     )
 
     $returnValue = @{
-        Ensure                    = [EnsureEnumeration]::Absent
+        Ensure                    = [EnsureEnumeration].GetEnumName(0)
```

Comment:
This seems like a reduction in readability of the code?

Reply:
Otherwise, the index of the enum value is returned

Reply:
I think you can use `([EnsureEnumeration]::Absent).ToString()`

Reply:
fixed